### ISSUE TITLE
FR27 Implement GIS database

### ISSUE THREAD

**Issue by gubuntu** [kartoza/LEDET_BIMS#33]
- a separate instance of PostGIS from the Django backend. This one is purely for vector 'GIS' layers.
- a bysync filesystem for raster data

**Comment by NyakudyaA** [comment]
The docker-compose utilizes the default geonode database for the Django tables and the kartoza/postgis:10 for spatial data so that we can leverage the replication.

**Comment by gubuntu** [comment]
temporary implementation on http://lbimsgis.kartoza.com

production implementation will be on a LEDET server

discuss with @dimasciput how to merge with the new integrated BIMS-GeoNode 

GIS layers must be :
- served by GeoServer / GeoNode 
- accessible to BIMS via OWS
- accessible to BIMS backend (e.g. boundaries for clustering)
- accessible to LEDET GIS users from QGIS
  - but GIS users must not be able to access BIMS backend not biodiversity records

So we need to decide soon how we separate databases, schemas etc.

**Comment by gubuntu** [comment]
@NyakudyaA test this against the testing server, then against the staging server once it has GeoNode and then against the production server once it is deployed there. 

Then ask Vincent and other LEDET staff to test on their LAN

**Comment by NyakudyaA** [comment]
Still waiting on the Geonode restore. Otherwise, I have a backup of the spatial database and django in the btsync share.

**Comment by gubuntu** [comment]
@NyakudyaA please size remaining work and update status in waffle

**Comment by NyakudyaA** [comment]
The database has been implemented. There is a btsync that can be used for raster data within the Geoserver container. @dimasciput is going to add the backup for the spatial database into the orchestration.

**Comment by gubuntu** [comment]
@NyakudyaA as per my comment amonth ago, we can only clear this once the production instance is deployed at LEDET

**Comment by NyakudyaA** [comment]
@dimas just generate a key for btsync and add it as a folder to geoserver

**Comment by gubuntu** [comment]
@NyakudyaA create this as a new ticket rather:

> @dimas just generate a key for btsync and add it as a folder to geoserver

**Comment by NyakudyaA** [comment]
Added a new ticket for this
